### PR TITLE
Icemule/update

### PR DESCRIPTION
**Pull request by BryanSchuetz**
More updates to Icemule theme.

### CODE REVIEW

File: src/app/styles/themes/icemule.scss [hunk 1]
```diff
@@ -78,151 +81,126 @@ pre {
...
+  font-size: 0.9rem;
+  .label {
+    text-transform: capitalize;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+    overflow: hidden;
+    overflow: hidden;
+    overflow: hidden;
+    overflow: hidden;
+    overflow: hidden;
```
Comment (chriscoyier) [inline src/app/styles/themes/icemule.scss:106]: lol

Comment (BryanSchuetz) [reply src/app/styles/themes/icemule.scss:106]: Hah, let me fix that.

**Comment by BryanSchuetz** [comment]
OK, removed a bunch of cruft. Should be ready for merging now. 

I guess at some point—if we're really wanting the themes to essentially just be changes to a color palette,  I'll have to trash this all together anyway.

**Comment by chriscoyier** [comment]
> I guess at some point—if we're really wanting the themes to essentially just be changes to a color palette, I'll have to trash this all together anyway.

I think moving in that direction is smart, but I'm not sure if it needs to get all the way to the point where there is no custom CSS whatsoever in a theme that isn't a custom property. I think the point is more like...

1. Themes should feel somewhat cohesive.
2. Integrating custom properties into the core theme setup is just a good idea anyway. It will make future themes, particularly simple ones, very easy to create.
3. It will get hard to refactor UI if there are a tons of themes doing very custom things. I'd rather not be beholden to themes to evolve the UI.

**Pull request merged**